meeko/cli/mk_prepare_ligand.py

```diff
@@ -551,6 +556,19 @@ def main():
     nr_failures = 0
     is_after_first = False
     preparator = MoleculePreparation.from_config(config)
+
+    if args.charge_from_prop is not None: 
+        if args.charge_model != "read": 
+            print('--charge_from_prop must be used with --charge_model "read"')
+            sys.exit(1)
+
+        preparator.charge_model = "read"
+        if args.charge_from_prop: 
+            preparator.charge_atom_prop = args.charge_from_prop[0]
+        else:
+            if ext=="mol2": 
+                preparator.charge_atom_prop = "_TriposPartialCharge"
```

Comment:
this is inside the `if args.charge_from_prop is not None` so i think this never gets executed